css/functions.json

```diff
@@ -557,5 +565,13 @@
     ],
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/var"
+  },
+  "view()": {
+    "syntax": "view([<axis> || <'view-timeline-inset'>]?)",
+    "groups": [
+      "CSS Animations"
+    ],
+    "status": "experimental",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/animation-timeline/view"
```

Comment:
Is this page incoming? It's currently a 404

Reply:
Yes, that's correct.